### DISCUSSION TITLE
I created exportify-cli

### DISCUSSION THREAD

**Discussion by donmerendolo** [pavelkomarov/exportify#59] [Announcements]
Hi! I wanted to let you know that I (well, ChatGPT) built a Python program called [exportify-cli](https://github.com/donmerendolo/exportify-cli), inspired by your repository, to back up Spotify playlists directly from the terminal (I wanted to automate the process). Just wanted to say thanks! 😃

**Comment by pavelkomarov** [comment]
Glad Chat could ingest my and others’ code lol.

I enabled discussions on this repo. Hoping these kinds of things end up there in the future. When I see "new issue", my heart skips a beat.
